src/init/starship.ps1

```diff
@@ -106,7 +106,13 @@ $null = New-Module starship {
         $arguments += "--status=$($lastExitCodeForPrompt)"
 
         # Invoke Starship
-        Invoke-Native -Executable ::STARSHIP:: -Arguments $arguments
+        $prompt = Invoke-Native -Executable ::STARSHIP:: -Arguments $arguments
```

Comment:
Please use a different name for this to avoid confusion with the `global:prompt` function.

Reply:
Any suggestions?